projects/lib-posts/src/lib/lib-posts.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+
+interface Post {
+  body: string;
+  id: number;
+  userId: number;
+  title: string;
+}
+
+@Component({
+  selector: 'posts-lib-posts',
+  templateUrl: './lib-posts.component.html',
+  styleUrls: ['./lib-posts.component.scss'],
+})
```

Comment:
what do you think about changeDetectionStrategy?

Reply:
aproove